lib/services/project-service.ts

```diff
@@ -66,6 +67,17 @@ export class ProjectService implements IProjectService {
 		this.$logger.printMarkdown("Project `%s` was successfully created.", projectName);
 	}
 
+	@exported("projectService")
+	public isValidNativeScriptProject(pathToProject?: string): boolean {
```

Comment:
Because if it is `null` or `undefined` the method will still work. It will check the current directory.